cg/cli/workflow/rnafusion/base.py

```diff
@@ -258,33 +258,33 @@ def metrics_deliver(context: CGConfig, case_id: str, dry_run: bool) -> None:
     except CgError as error:
         raise click.Abort() from error
 
-    if not analysis_api.trailblazer_api.is_latest_analysis_qc(case_id=case_id):
-        LOG.error("The analysis status must be in the QC step to be stored.")
-        raise click.Abort()
-    analysis_api.write_metrics_deliverables(case_id=case_id, dry_run=dry_run)
-    if dry_run:
-        LOG.info("Dry-run: QC metrics validation would be performed.")
-        return
-    try:
-        LOG.info("Validating QC metrics.")
-        analysis_api.validate_qc_metrics(case_id=case_id)
-    except MetricsQCError as error:
-        LOG.error(f"QC metrics failed for {case_id}")
-        analysis_api.trailblazer_api.set_analysis_status(
-            case_id=case_id, status=AnalysisStatus.FAILED
-        )
-        analysis_api.trailblazer_api.add_comment(case_id=case_id, comment=str(error))
-        raise click.Abort() from error
-    except CgError as error:
-        LOG.error(f"Could not create metrics deliverables file: {error}")
+    if analysis_api.trailblazer_api.is_latest_analysis_qc(case_id=case_id):
+        analysis_api.write_metrics_deliverables(case_id=case_id, dry_run=dry_run)
+        if dry_run:
+            LOG.info("Dry-run: QC metrics validation would be performed.")
+            return
+        try:
+            LOG.info("Validating QC metrics.")
```

Comment:
Dot or no dot? 🤯 
```suggestion
            LOG.info("Validating QC metrics")
```

Reply:
This choice is quite arbitrary. We have been using it without a dot in `cg`, but there is no specific reason. I would take away the dot and leave that discussion for later.